tests/test_scaler.py

```diff
@@ -298,3 +298,40 @@ def get(url, version, **kwargs):
           exclude_namespaces=[], exclude_deployments=[], exclude_statefulsets=[], dry_run=False, grace_period=300, downtime_replicas=0)
 
     api.patch.assert_not_called()
+
+
+def test_scaler_namespace_annotation_replicas(monkeypatch):
+    api = MagicMock()
+    monkeypatch.setattr('kube_downscaler.scaler.helper.get_kube_api', MagicMock(return_value=api))
+    SCALE_TO = 3
+
+    def get(url, version, **kwargs):
+        if url == 'pods':
+            data = {'items': []}
+        elif url == 'deployments':
+            data = {'items': [
+                {
+                    'metadata': {
+                        'name': 'deploy-1', 'namespace': 'default', 'creationTimestamp': '2019-03-01T16:38:00Z',
+                    }, 'spec': {'replicas': 5}
+                },
+                ]}
+        elif url == 'namespaces/default':
+            data = {'metadata': {'annotations': {'downscaler/downtime-replicas': '3'}}}
```

Comment:
nit: you can re-use the `SCALE_TO` variable here (instead of "3")

Reply:
fixed